charts/ix-chart/2212.0.1/questions.yaml

```diff
@@ -427,7 +426,6 @@ questions:
                 schema:
                   type: int
                   required: true
-                  min: 9000
```

Comment:
This is not required ? 
Node port services are not used with host networking iirc ?

Reply:
You are correct. It's not even showing when hostNet is enabled.